lib/build/bundle.js

```diff
@@ -220,15 +238,21 @@ exports.Bundle = class {
       if (includeBundles) {
         bundleMetadata[currentBundle.moduleId] = currentBundle.getBundledModuleIds();
       }
-
-      config.paths[currentBundle.moduleId] = location + '/' + currentBundle.moduleId;
+      config.paths[currentBundle.moduleId] = location + '/' + currentBundle.moduleId + (buildOptions.rev && currentBundle.hash ? '-' + currentBundle.hash : '');
     }
 
     if (includeBundles) {
       config.bundles = bundleMetadata;
     }
 
-    return 'requirejs.config(' + JSON.stringify(config) + ');'
+    return 'function _loadAureliaRequireJSConfig(){requirejs.config(' + JSON.stringify(config) + ');}'
```

Comment:
Also, the actual code `requirejs.config`... should be extracted to another method. As mentioned above, we'll ultimately have several different loaders we need to support. Each is going to generate their configuration a bit differently.